prov/gni/src/gnix_util.c

```diff
@@ -243,6 +243,36 @@ static int gnix_alps_init(void)
 	return ret;
 }
 
+typedef struct ccm_alps_info {
+	uint32_t version;
+	uint8_t ptag;
+	uint32_t cookie;
+} ccm_alps_info_t;
+
+int get_ccm_ptag_cookie(char *filename)
+{
+	int rc, fd;
+	uint32_t version;
+	ccm_alps_info_t info;
+	GNIX_INFO(FI_LOG_FABRIC, "Reading job info file %s", filename);
+
+	fd = open(filename, O_RDONLY);
+	if (fd < 0)
+		return -errno;
```

Comment:
I ran into this issue with the kdreg stuff.  Any thoughts on what to do generally speaking?  Not all the possible errnos are defined ad FI_E* codes.  I put in FI_WARN's with strerror(), to help with that, but it's still not great.